schema/deploy/computed_columns/project_manager_names_concatenated.sql

```diff
@@ -0,0 +1,31 @@
+-- Deploy cif:computed_columns/project_manager_names_concatenated to pg
+
+-- instead returns text, concatenated, it'll be an empty string if there are no managers. Then it wouldn't need to take an argument, filter for empty string and not null (check filters for null or empty options). includes insensitive empty string (potentially consider refactoring to use tvector later)
+
+begin;
+
+create or replace function cif.project_manager_names_concatenated(cif.project)
```

Comment:
I think `cif.project_primary_managers` is a more explicit name for this function
```suggestion
create or replace function cif.project_primary_managers(cif.project)
```